internal/livelint/check_logs_internal_test.go

```diff
@@ -0,0 +1,159 @@
+package livelint
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/matryer/is"
+	apiv1 "k8s.io/api/core/v1"
+	typedapiv1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	restclient "k8s.io/client-go/rest"
+	fake "k8s.io/client-go/rest/fake"
+)
+
+type LogResponse struct {
```

Comment:
This should not be exported.

```suggestion
type logResponse struct {
```